book_config.py

```diff
@@ -195,8 +189,13 @@ def search_for_goodreads_url(self):
             self.goodreads_url_edit.setText('')
 
     def update_aliases(self):
+        if 'goodreads.com' not in self.goodreads_url_edit.text():
```

Comment:
doing urlsplit on 'goodreads.com/test' doesn't put 'www.goodreads.com' in netloc